### PR TITLE
feat: add EEF SSL options

### PR DESCRIPTION
**Pull request by davidjulien**

### CODE REVIEW

File: lib/ex_ssl_options.ex [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
-  Documentation for `ExSslOptions`.
+  This module returns SSL options
   """
 
   @doc """
-  Hello world.
+  SSL options according to [EEF Security WG recommandations](https://github.com/erlef/security-wg/blob/master/docs/secure_coding_and_deployment_hardening/ssl.md).
+  """
+  def eef_options do
+    preferred_ciphers = [
```
Comment (StephaneRob) [inline lib/ex_ssl_options.ex:10]: unfortunately sometimes we need to deal with weaker cipher suites (ex: jobs importer).

**Review by StephaneRob — APPROVED** [review]

File: lib/ex_ssl_options.ex (hunk 1 above)
Comment (davidjulien) [reply lib/ex_ssl_options.ex:10]: I can do a version including the 2 disabled ciphers. I thought to add different functions if we found other recommandations or constraints.

As for the weak cypher, they said that _Recent versions of Erlang/OTP disable most weak, legacy SSL/TLS protocol versions and cipher suites._ Thus if other servers really use weak cyphers, we won't be able to connect to them, even without restricting the cyphers list.

Comment (davidjulien) [reply lib/ex_ssl_options.ex:10]: I will do another PR with only required options

**Review by scollon42 — APPROVED** [review]
Nice !

**Pull request merged**